backend/src/main/java/com/eshka/entity/Subject.java

```diff
@@ -20,4 +20,6 @@
     private String title;
     @Column(name = "description")
     private String description;
+    @Column(name = "teamId")
+    private long teamId;
```

Comment:
I would like to add some notes. `Subject` is an entity class. It means you should use the whole `Team` object instead of a simple team id.
Also, you should specify the relationship in the database for this object. For one team we have many subjects. So, the relationship, in this case, is 'many to one'. Please, add the corresponding annotation to team field of the class. It should look like this:
```java
@ManyToOne(fetch = FetchType.LAZY, cascade = {CascadeType.MERGE, CascadeType.PERSIST})
@JoinColumn(name = "team_id")
private Team team;
```